### PR TITLE
[batch2] add batch2 check, delint

### PR DESCRIPTION
**Pull request by cseed**
Added batch2_check step in build, fixed lint errors.

### CODE REVIEW

**Comment by cseed** [comment]
Also, disabled uvloop until it is a bit easier to set it up before importing anything else.  pylint requires imports to come before all code.

**Comment by danking** [comment]
I like this, `uvloop.install()` has always made me a bit unsettled.

File: batch2/batch/worker.py [hunk 1]
```diff
@@ -348,6 +339,8 @@ def __init__(self, worker, parameters, cpu_sem):
 
         self._run_task = asyncio.ensure_future(self.run(cpu_sem))
 
+        self.last_updated = None
```
Comment (jigold) [inline batch2/batch/worker.py]: Was there something you wanted to use this variable for?

Comment (cseed) [reply batch2/batch/worker.py]: The linter gives an error if a member is assigned in another method but not in __init__.  I guess there is a danger, if you call the methods out of order, that you get an attribute error unexpectedly.

Comment (danking) [reply batch2/batch/worker.py]: FWIW, I find defining all the fields in the `__init__` method helpful when reading unfamiliar files. Especially given the lack of any type information.

**Pull request merged**